Referencing NETStandard.Library in a Library project causes build failure

### Stack trace

```
\.nuget\packages\netstandard.library\2.0.0-beta-25008-01\build\NETStandard.Library.targets(108,5): error MSB4018: The "RemoveDepsFileConflicts" task failed unexpectedly.\r [\src\Microsoft.Extensions.CommandLineUtils\Microsoft.Extensions.CommandLineUtils.csproj]
\.nuget\packages\netstandard.library\2.0.0-beta-25008-01\build\NETStandard.Library.targets(108,5): error MSB4018: System.IO.FileNotFoundException: Could not find file '\src\Microsoft.Extensions.CommandLineUtils\bin\Debug\net451\Microsoft.Extensions.CommandLineUtils.deps.json'.\r [\src\Microsoft.Extensions.CommandLineUtils\Microsoft.Extensions.CommandLineUtils.csproj]
\.nuget\packages\netstandard.library\2.0.0-beta-25008-01\build\NETStandard.Library.targets(108,5): error MSB4018: File name: '\src\Microsoft.Extensions.CommandLineUtils\bin\Debug\net451\Microsoft.Extensions.CommandLineUtils.deps.json'\r [\src\Microsoft.Extensions.CommandLineUtils\Microsoft.Extensions.CommandLineUtils.csproj]
\.nuget\packages\netstandard.library\2.0.0-beta-25008-01\build\NETStandard.Library.targets(108,5): error MSB4018:    at System.IO.Win32FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, FileStream parent)\r [\src\Microsoft.Extensions.CommandLineUtils\Microsoft.Extensions.CommandLineUtils.csproj]
\.nuget\packages\netstandard.library\2.0.0-beta-25008-01\build\NETStandard.Library.targets(108,5): error MSB4018:    at System.IO.Win32FileSystem.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, FileStream parent)\r [\src\Microsoft.Extensions.CommandLineUtils\Microsoft.Extensions.CommandLineUtils.csproj]
\.nuget\packages\netstandard.library\2.0.0-beta-25008-01\build\NETStandard.Library.targets(108,5): error MSB4018:    at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)\r [\src\Microsoft.Extensions.CommandLineUtils\Microsoft.Extensions.CommandLineUtils.csproj]
\.nuget\packages\netstandard.library\2.0.0-beta-25008-01\build\NETStandard.Library.targets(108,5): error MSB4018:    at System.IO.File.Open(String path, FileMode mode, FileAccess access, FileShare share)\r [\src\Microsoft.Extensions.CommandLineUtils\Microsoft.Extensions.CommandLineUtils.csproj]
\.nuget\packages\netstandard.library\2.0.0-beta-25008-01\build\NETStandard.Library.targets(108,5): error MSB4018:    at Microsoft.DotNet.Build.Tasks.RemoveDepsFileConflicts.Execute()\r [\src\Microsoft.Extensions.CommandLineUtils\Microsoft.Extensions.CommandLineUtils.csproj]
\.nuget\packages\netstandard.library\2.0.0-beta-25008-01\build\NETStandard.Library.targets(108,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()\r [\src\Microsoft.Extensions.CommandLineUtils\Microsoft.Extensions.CommandLineUtils.csproj]
\.nuget\packages\netstandard.library\2.0.0-beta-25008-01\build\NETStandard.Library.targets(108,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__25.MoveNext() [\src\Microsoft.Extensions
```

Looks like the target attempts to reads a deps file which isn't generated for `<OutputType>Library</OutputType>`. We're using the `1.0.0-rc4-004777` SDK.

--- a/netstandard/tools/targets/NETStandard.Library.targets
+++ b/netstandard/tools/targets/NETStandard.Library.targets
@@ -104,7 +104,8 @@
   </PropertyGroup>
   <UsingTask TaskName="RemoveDepsFileConflicts" AssemblyFile="$(NETStandardToolsTaskDirectory)NETStandard.Tools.dll" />
   <Target Name="RemoveDepsFileConflicts" AfterTargets="$(RemoveDepsFileConflictsAfter)"
-          Inputs="$(ProjectDepsFilePath)" Outputs="$(_RemoveDepsFileConflictsSemaphore)">
+          Inputs="$(ProjectDepsFilePath)" Outputs="$(_RemoveDepsFileConflictsSemaphore)"
+          Condition="'$(GenerateDependencyFile)'=='true'">
     <RemoveDepsFileConflicts DepsFilePath="$(ProjectDepsFilePath)" Conflicts="@(_ConflictPackageFiles)" />
 
     <Touch Files="$(_RemoveDepsFileConflictsSemaphore)" AlwaysCreate="true">

